app-config.yaml

```diff
@@ -48,37 +47,45 @@ backend:
   # The production database configuration is stored in app-config.production.yaml
   database:
     client: better-sqlite3
-    connection: ':memory:'
+    connection: ":memory:"
     # client: pg
     # connection:
     #   host: localhost #${POSTGRES_HOST}
     #   port: 5432
     #   user: postgres # ${POSTGRES_USER}
     #   password: secret # ${POSTGRES_PASSWORD}
-    # plugin: 
+    # plugin:
     #   catalog:
     #     connection:
     #       database: postgres
-      # app:
-      #   connection:
-      #     database: postgres
+    # app:
+    #   connection:
+    #     database: postgres
   cache:
     store: memory
 
-      
   # workingdirectory: /tmp # use this to configure a working directory for the scaffolder, defaults to the os temp-dir
 
 integrations:
   github:
     - host: github.com
       token: ${GITHUB_TOKEN}
       # apps:
-        # - $include: github-app-backstage-projectx-test1-credentials.yaml
+      # - $include: github-app-backstage-projectx-test1-credentials.yaml
 
 proxy:
-  '/test':
-    target: 'https://example.com'
+  "/test":
+    target: "https://example.com"
     changeorigin: true
+
+  "/grafana/api":
+    target: https://backstage.devstats.cncf.io
```

Comment:
Why are we pointing to devstats.cncf's backstage instance here?

Reply:
just used as an instance open path to fulfill the request, we will replace with our own hosting.